doc/compiled.json

```diff
@@ -6908,7 +6908,9 @@
             "schema": {
               "type": "object",
               "properties": {}
-            }
+            },
+            "style": "deepObject",
+            "explode": true
```

Comment:
I checked locally and without these options as well, the test cases pass. Can be removed unless we decide to make any alterations or changes to the currently proposed solution.